### ISSUE TITLE
Examples should be renamed to include simpletest

### ISSUE THREAD

**Issue by kattni** [adafruit/Adafruit_CircuitPython_Waveform#12]
1. Example `waveform_sine_demo.py` should be renamed to `waveform_sine_simpletest.py`. 
2. Example `waveform_square_demo.py` should be renamed to `waveform_square_simpletest.py`. 

This will fit with CircuitPython library standards.

The lines including this file name in the `docs/examples.rst` file must also be updated to reflect the name change, e.g.:
```
.. literalinclude:: ../examples/current_file_name.py
    :caption: examples/current_file_name.py
    :linenos:
```
would be updated to:
```
.. literalinclude:: ../examples/current_file_name_changed.py
    :caption: examples/current_file_name_changed.py
    :linenos:
```

**Comment by abkosar** [comment]
Hey! I would like to work on this!

**Comment by kattni** [comment]
@abkosar Sounds good!

**Comment by kattni** [comment]
Fixed by #13
